Weather/Weather/Scenes/ListCityWeather/ListCityWeatherViewController.swift

```diff
@@ -102,7 +103,9 @@ extension ListCityWeatherViewController: UICollectionViewDataSource, UICollectio
     }
     
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        pushViewController()
+        if presentedViewController == nil {
```

Comment:
```suggestion
        if let _ = presentedViewController {
```
cái này trông có vẻ hợp lý hơn,
nhưng ko sửa cũng ksao nhé

Reply:
Thank đông :3